tawk_to.admin.inc

```diff
@@ -289,11 +289,17 @@ function tawk_to_admin_form_submit($form, &$form_state) {
     switch ($column) {
       case 'hide_oncustom':
       case 'show_oncustom':
-        // replace newlines and returns with comma, and convert to array for saving
-        $value = str_replace(["\r\n", "\r", "\n"], ',', $value);
-        $value = explode(',', $value);
-        $value = (empty($value)||!$value)?array():$value;
-        $jsonOpts[$column] = json_encode($value);
+        // split by newlines, then remove empty lines
+        $value = str_replace("\r", "\n", $value);
```

Comment:
no `urldecode()` here because form is not serialized, actual string is posted via form action
![Screenshot from 2024-07-23 16-42-21](https://github.com/user-attachments/assets/7684b6d8-2b6f-4bf4-8b14-2b2ce2df9baa)